Add explicit note of relative path to packit's `create_archive` (#151)
packit/packit.dev: content/docs/actions.md

|        | `create-patches`      | upstream git repo | after sync of upstream files to the downstream                                    | replace patching                          |
|        | `fix-spec-file`            | upstream git repo | after creation of a tarball and before running rpmbuild command                   | this action changes spec file to use the new tarball                          |

> **create-archive** - is expected to return the created archive name (in case of a path, must be relative). If there are more steps, then one of them has to return the archive name.

Reviewer: ```suggestion
**create-archive** - is expected to return a relative path to the archive - relative within the repository. If there are more steps, then one of them has to return the archive name.
```
